### ISSUE TITLE
Check terminology

### ISSUE THREAD

**Issue by tedhabermann** [opengeospatial/geotiff#28]

**Comment by tedhabermann** [comment]
I updated the terminology using the 20180117 version of the document as a guide. It appears that Roger started with an older version of the document when he made his comments. **These changes are in the master branch.**

There are a few questions that came up as I did this:
1. Formattiing: Word template had numbers for each term - they are not in the ASCIIDoc. In addition a few of the standards on line that I checked had the terms in italics. **Do we need to do that?**
1. It appears that Reese identified some terms that he thought were close to the ones we are using and added definitions for those. If these work - great - if not, we need to add definitions for the original terms. The ones I identified were:
datum -> reference frame
ellipsoid -> reference ellipsoid
projected coordinate system -> projected coordinate reference system
raster ->raster space
1. the definition for projection does not look correct.
